planetiler-core/src/main/proto/stream_archive_proto.proto

```diff
@@ -19,7 +18,6 @@ message TileEntry {
 }
 
 message InitializationEntry {
-  Metadata metadata = 1;
```

Comment:
@bbilger I need to remove metadata from the archive init call since it's not fully known until done writing all tiles. Any concerns about this?

Reply:
no concerns :+1: 

just a note for the future: once 1.0 gets out, we might want to consider to ensure that changes in protobuf are backwards compatible - which in that case would mean to reserve the field number+name
but absolutely not needed now (pre 1.0), and I don't think anybody is using it, anyways